### ISSUE TITLE
[SUGGESTION] Autopilot for space ships and being able to walk around while moving

### ISSUE THREAD

**Issue by N3XTGEN82** [tsunamayo/Starship-EVO#3035]
It would be really good if you could walk around inside your ship while it is moving without someone else piloting it, like space engineers. The little switch on the yolk maybe you can interact with yolk to set it on autopilot without being in control at the same time. Especially helps when you have no one else to play with as crew. Unless A.I crew is coming eventually/planned.

**Comment by Kaiser-Indrasil** [comment]
Just a quick heads up.
This is yolk:
![image](https://user-images.githubusercontent.com/63756057/95094021-56697880-0721-11eb-891d-2454e57ce81f.png)
This is yoke:
![image](https://user-images.githubusercontent.com/63756057/95094113-6f722980-0721-11eb-86f8-3fe409fe2ccb.png)
And yes, AI crew is planned.

**Comment by tsunamayo** [comment]
Yes it make sense, actually it worked like that before. I will put this back in the game.
Damn this yoke looks tasty...
Cheers

**Comment by WILDCATreactor** [comment]
>didn't paste the best yoke of all
LMFTFY

![steel battalion controller](https://user-images.githubusercontent.com/70435732/95646413-4392d300-0a96-11eb-9ec2-bb1eff4b6b69.jpg)
